docs/BackupNoSQL.md

```diff
@@ -0,0 +1,132 @@
+# Back up a NoSQL database in a Kubernetes environment
+
+This guide explains how to create a transactionally consistent backup of managed databases that ScalarDB or ScalarDL uses in a Kubernetes environment. Please note that, when using a NoSQL database or multiple databases, you **must** pause ScalarDB or ScalarDL when creating a transactionally consistent backup.
```

Comment:
```suggestion
This guide explains how to create a transactionally consistent backup of managed databases that ScalarDB or ScalarDL uses in a Kubernetes environment. Please note that, when using a NoSQL database or multiple databases, you **must** pause ScalarDB or ScalarDL to create a transactionally consistent backup.
```
Isn't it unnatural to use `when` multiple times in one sentence? (> @josh-wong )

Reply:
My mistake. Your change (`to create`) is better in this case.

Reply:
Thank you for your confirmation! I will apply the suggestion.